minorminer/utils/parallel_embeddings.py

```diff
@@ -0,0 +1,630 @@
+            provided, it is generated as a fully yielded square sublattice
+            of `T`, with `m=sublattice_size`.
+            If `tile==S`, `embedder` can be ignored since a 1:1 mapping is
+            necessary on each subgraph and easily verified by checking the
+            mask is complete.
+        sublattice_size: Parameterizes the tile when it is not provided
+           as an input argument: defines the number of rows and columns
+           of a square sublattice (parameter m of the dwave_networkx graph
+           family matching T).
+           :code:`lattice_size_lower_bound()`
+           provides a lower bound based on a fast feasibility filter.
+        max_num_emb: Maximum number of embeddings to find.
+            Defaults to inf (unbounded).
+        use_filter: Specifies whether to check feasibility of arguments for
+            embedding independently of the embedder routine. Defaults to False.
+        embedder: Specifies the embedding search method, a callable taking S, T as
+            the first two arguments. Defaults to minorminer.subgraph.find_subgraph.
+        embedder_kwargs: Dictionary specifying arguments for the `embedder`
+            other than `S`, `T`.
+        one_to_iterable: Specifies whether the embedder returns a dict with
+            iterable values. Defaults to False to match find_subgraph.
+        shuffle_all_graphs: If True, the tile-masked target graph and source graph are
+            shuffled on each embedding attempt. Note that, if the embedder supports
+            randomization this should be preferred by use of embedder_kwargs. If an
+            embedder is a function of the node or edge order this allows
+            diversification of the embeddings found.
+        shuffle_sublattice_order: If True the ordering of sublattices (displacement
+            of the tile (mask) on the target graph is randomized. This can allow
+            for diversification of the embeddings found.
+        seed: seed for the `numpy.random.Generator` controlling shuffling (if
+            invoked).
+
+    Raises:
+        ValueError: If the target graph `T` is not of type zephyr, pegasus, or
+            chimera.
+
+    Returns:
+        list: A list of disjoint embeddings.
+    """
+    if sublattice_size is None and tile is None:
+        return find_multiple_embeddings(
+            S=S,
+            T=T,
+            max_num_emb=max_num_emb,
+            seed=seed,
+            embedder=embedder,
+            embedder_kwargs=embedder_kwargs,
+        )
+
+    if use_filter and tile is None:
+        feasibility_bound = lattice_size_lower_bound(
+            S=S, T=T, one_to_one=not one_to_iterable
+        )
+        if feasibility_bound is None or sublattice_size < feasibility_bound:
+            warnings.warn("sublattice_size < lower bound: embeddings will be empty.")
+            return []
+    # A possible feature enhancement might allow for sublattice_size (m) to be
```

Comment:
I'd suggest adding in a few empty lines, e.g., after returns or if-else blocks, for clarity.
```suggestion
            return []

    # A possible feature enhancement might allow for sublattice_size (m) to be
```